### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by alitariq4589**
- Updated qemu documentation: Updated qemu command for enabling port forwarding from host machine to qemu VM. This will enable reader to understand how to do SSH inside qemu from outiside network.  
- Added GitLab integration documentation

### CODE REVIEW

**Review by mmhus — CHANGES_REQUESTED** [review]
Change 2222 to say a random number

**Comment by alitariq4589** [comment]
> Change 2222 to say a random number  

I have changed it to generic names now. `<host_port>` and `<VM_port>`

File: mkdocs_documentation/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mmhus) [inline mkdocs_documentation/.vscode/settings.json:1]: Remove the vscode folder, also how many docs did you add?

Comment (alitariq4589) [reply mkdocs_documentation/.vscode/settings.json]: I have removed `mkdocs_documentation` folder. I added `jenkins_gitlab_integration` document.

**Review by mmhus — CHANGES_REQUESTED** [review]
I can't find the associated changes in the .md files. Please re-check.

**Comment by alitariq4589** [comment]
> I can't find the associated changes in the .md files. Please re-check.

There are no md files. These files are generated by mkdocs. md files were part of `mkdocs_documentation` folder which I deleted after building mkdocs and creating pull request.

**Review by mmhus — APPROVED** [review]

**Pull request merged**